SpatialGDK/Source/SpatialGDKEditorToolbar/Private/SpatialGDKEditorToolbar.cpp

```diff
@@ -1047,6 +1074,10 @@ void FSpatialGDKEditorToolbarModule::OnPropertyChanged(UObject* ObjectBeingModif
 		{
 			OnAutoStartLocalDeploymentChanged();
 		}
+		else if(PropertyName.ToString() == TEXT("bStartLocalServerWorker"))
```

Comment:
```suggestion
		else if (PropertyName.ToString() == GET_MEMBER_NAME_CHECKED(USpatialGDKEditorSettings, bStartLocalServerWorker))
```
It would also be awesome if the two ifs above could use this pattern (this makes it so the code won't compile if the property name changes, so it's harder to accidentally break it)